tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -6433,20 +6433,29 @@ HRESULT TableParameterHandler::ParseTableRow() {
   return S_OK;
 }
 
-static bool CompareOutputWithExpectedValueInt(int output, int ref, int tolerance) {
-    return output - ref <= tolerance && ref - output <= tolerance;
+static bool CompareOutputWithExpectedValueInt(int output, int ref,
+                                              int tolerance) {
+  return output - ref <= tolerance && ref - output <= tolerance;
```

Comment:
Could we change to ((output - ref) <= tolerance) && ((ref - output) <= tolerance); ?
It is hard to follow now.